Source/WebCore/page/Page.cpp

```diff
@@ -1863,7 +1863,7 @@ void Page::updateRendering()
 
     auto runProcessingStep = [&](RenderingUpdateStep step, const Function<void(Document&)>& perDocumentFunction) {
         m_renderingUpdateRemainingSteps.last().remove(step);
-        forEachDocument(perDocumentFunction);
+        forEachRenderableDocument(perDocumentFunction);
```

Comment:
I hope using `forEachRenderableDocument` in all these places isn't a perf regression.